.github/workflows/dist.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   fedora:
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
ftr, if you don't build this with every push or PR, you won't find out that it's broken until after the tag is pushed. And by then it's too late, because fixing the spec file would require you to re-tag and that's not really worth it.